### PR TITLE
Update content.md

### PR DESCRIPTION
**Pull request by martab1994**
Fix typos and link

## What This PR Changes
- (Please explain here why you created the pull request and specify what it changes)

## Contribution Guidelines
- [ ] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/04.pro/boards/portenta-h7/tutorials/getting-started-openmv-micropython/content.md [hunk 1]
```diff
@@ -38,13 +38,13 @@ The OpenMV IDE is meant to provide an Arduino like experience for simple machine
...
 
 ### 1. Downloading the OpenMV IDE
 
-Before you can start programming OpenMV scripts for the Portenta you need to download and install the OpenMV IDE.
+Before you can start programming OpenMV scripts for the Portenta, you need to download and install the OpenMV IDE.
 
-***IMPORTANT: Please make sure to update the bootloader to the most recent version to benefit from the latest improvements. Follow [these steps](/tutorials/portenta-h7/updating-the-bootloader) before you proceed with the next step of this tutorial.***
+***IMPORTANT: Please make sure to update the bootloader to the most recent version to benefit from the latest improvements. Follow [these steps](https://docs.arduino.cc/tutorials/portenta-h7/updating-the-bootloader) before you proceed with the next step of this tutorial.***
```
Comment (marqdevx) [inline content/hardware/04.pro/boards/portenta-h7/tutorials/getting-started-openmv-micropython/content.md:47]: The previous link is working tho

File: content/hardware/04.pro/boards/portenta-h7/tutorials/getting-started-openmv-micropython/content.md [hunk 2]
```diff
@@ -162,14 +162,14 @@ In this tutorial you learned how to use the OpenMV IDE with your Portenta board.
 ### Next Steps
 
 -   Experiment with MicroPythons capabilities. If you want some examples of what to do, take a look at the examples included in the OpenMV IDE. Go to: **File > Examples > Arduino > Portenta H7** in the OpenMV IDE.
--   Take a look at our other Portenta H7 tutorials which showcase its many uses. You can find them [here](https://docs.arduino.cc/hardware/portenta-h7#tutorials).
+-   Take a look at your other Portenta H7 tutorials which showcase its many uses. You can find them [here](https://docs.arduino.cc/hardware/portenta-h7#tutorials).
```
Comment (marqdevx) [inline content/hardware/04.pro/boards/portenta-h7/tutorials/getting-started-openmv-micropython/content.md:165]: I see `our` better than `your`, but we can switch it with `the` better?

File: content/hardware/04.pro/boards/portenta-h7/tutorials/getting-started-openmv-micropython/content.md (hunk 1 above)
Comment (martab1994) [reply content/hardware/04.pro/boards/portenta-h7/tutorials/getting-started-openmv-micropython/content.md:47]: For me it was not working!

File: content/hardware/04.pro/boards/portenta-h7/tutorials/getting-started-openmv-micropython/content.md (hunk 2 above)
Comment (martab1994) [reply content/hardware/04.pro/boards/portenta-h7/tutorials/getting-started-openmv-micropython/content.md:165]: Definetely!

**Review by marqdevx — APPROVED** [review]
LGTM

File: content/hardware/04.pro/boards/portenta-h7/tutorials/getting-started-openmv-micropython/content.md (hunk 2 above)
Comment (marqdevx) [reply content/hardware/04.pro/boards/portenta-h7/tutorials/getting-started-openmv-micropython/content.md]: closing this, change addressed

File: content/hardware/04.pro/boards/portenta-h7/tutorials/getting-started-openmv-micropython/content.md (hunk 1 above)
Comment (marqdevx) [reply content/hardware/04.pro/boards/portenta-h7/tutorials/getting-started-openmv-micropython/content.md:47]: closing this, it is fine

**Pull request merged**